src/utlist.h

```diff
@@ -775,6 +836,37 @@ do {
  (head) = (add);                                                                               \
 } while (0)
 
+#define CDL_INSERT_INORDER(head,add,cmp)                                                       \
```

Comment:
Shouldn't there be a `CDL_INSERT_INORDER2` for the benefit of lists with non-default next and prev fields?